dll/cpl/mmsys/lang/en-US.rc

```diff
@@ -179,7 +179,7 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUPWINDOW | WS_VISIBLE | WS_CAPTION
 CAPTION "Multichannel"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "Description goes here.", 9506, 8, 4, 212, 24
+    LTEXT "By moving the sliders below you can adjust the speakers volume of all speakers connected to this computer.", 9506, 8, 4, 212, 24
```

Comment:
Ok first I checked whether this description may be a dynamic label that can be different depending on drivers and such, which seems to be not the case. I have addressed it in a separate commit c4bd0db0, which has reset translations for all languages. I will rebase this PR so there is no merge conflict.